h/assets.py

```diff
@@ -132,6 +134,22 @@ def _load_bundles(fp):
 assets_client_view = _add_cors_header(assets_client_view)
 
 
+def svg_icon(request, name, css_class=''):
```

Comment:
There are a few things that should be fixed here:

- This probably shouldn't be a method on the `request` object, since it isn't tied to a request in any way
- If the SVG has a `<title>` tag, that gets used as the tooltip for the icon. We should either strip those out when building the SVGs or remove them here
- Caching of the parsed XML content is missing